src/jailer/src/env.rs

```diff
@@ -626,6 +658,12 @@ impl Env {
                 println!("MMDS version 2 will not be available to use.");
             });
 
+        // If we have a minor version for /dev/userfaultfd the device is present on the host.
+        // Expose the device in the jailed environment.
```

Comment:
It is unlikely, but the file might not exist but the device might be loaded (because for example nothing ran `mknod`).
I thought this is more generic.

Reply:
In my mind checking `/dev/userfaultfd` and then moving it into the jail is more strait forward and makes code more simple (`std::path::Path::new("/dev/userfaultfd").exists()` vs opennig a file and parsing it's content).
Also I don't think there can be a scenario when `/dev/userfaultfd` does not exist, but the device is still available.